src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -148,6 +148,14 @@
   @Parameter(property = "user")
   private String user;
 
+  /** Number of retries for failing pushes, defaults to 3 */
+  @Parameter(property = "retryCount", defaultValue = "3")
+  private int retryCount;
```

Comment:
if this is only used for pushes, can you name the field (and the property) something like `retryPushCount` instead to make that clear?